src/test/ui/rfcs/rfc-2396-target_feature-safe-calls.rs

```diff
@@ -0,0 +1,19 @@
+#[target_feature(enable = "sse2")]
+fn foo() {}
+
+#[target_feature(enable = "sse2")]
+fn bar() {
+    foo();
+}
+
+#[target_feature(enable = "avx")]
+fn baz() {
+    foo(); //~ ERROR call to function with #[target_feature] is unsafe
+}
+
+fn main() {
+    unsafe {
+        bar();
+        baz();
+    }
+}
```

Comment:
In a separate test, let's also exercise `#[target_feature]` on functions in trait definitions and in trait impls.